po/LINGUAS

```diff
@@ -1 +1,2 @@
-de
\ No newline at end of file
+de
+nl
```

Comment:
I guess a newline counts as whitespace, but I'd rather have these space seperated

Reply:
Ah, normally devs want a new line there, but if you'd rather have a whitespace, I'll arrange that. 🙂